rotkehlchen/globaldb/handler.py

```diff
@@ -1203,6 +1225,86 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
              'to_timestamp': entry[3],
              } for entry in query]
 
+    @staticmethod
+    def rebuild_assets_list(user_db_conn: sqlite3.Connection) -> Tuple[bool, str]:
+        """
+        Delete all custom asset entries and repopulate from the last
+        builtin version
+        """
+        root_dir = Path(__file__).resolve().parent.parent
+        builtin_database = root_dir / 'data' / 'global.db'
+        detach_database = 'DETACH DATABASE "clean_db";'
+
+        connection = GlobalDBHandler()._conn
+        cursor = connection.cursor()
+        user_db_cursor = user_db_conn.cursor()
+
+        try:
+            # First chech that the operation can be made. For this create a set with the
+            # identifiers on the user db and the db packaged and make a difference. If the
+            # difference is not the empty set the operation is dangerous and the user should
+            # be notified.
+            query = cursor.execute('SELECT identifier from assets;')
+            user_ids = set(query.fetchall())
+            # Atach the clean db packaged with rotki
+            cursor.execute(f'ATTACH DATABASE "{builtin_database}" AS clean_db;')
+            # Get built in identifiers
+            query = cursor.execute('SELECT identifier from clean_db.assets;')
+            shipped_ids = set(query.fetchall())
+            diff_ids = user_ids - shipped_ids
+            if len(diff_ids) != 0:
+                # If we have non common elements we have to see if this assets are used anywhere
+                # in the database tables
+                diffs = ', '.join([f'"{id[0]}"' for id in diff_ids])
+                for table, column in TABLES_WITH_ASSETS:
+                    diffs_query = f'SELECT COUNT(*) FROM {table} where {column} in ({diffs})'
+                    user_db_cursor.execute(diffs_query)
+                    amount = user_db_cursor.fetchone()
+                    if amount[0] != 0:
+                        cursor.execute(detach_database)
+                        log.error(
+                            f'Assets inserted by the user are used in the database '
+                            f'tables {diff_ids}',
```

Comment:
Does this not fit in one line?

```suggestion
                            f'Assets inserted by the user are used in the database '
                            f'tables {diff_ids}',
```

Reply:
And so in my test I hit this, with a testing uniswap account.

```
[15/08/2021 00:22:39 CEST] ERROR rotkehlchen.globaldb.handler: Assets inserted by the user are used in the database tables {('_ceth_0xfAd45E47083e4607302aa43c65fB3106F1cd7607',), ('_ceth_0x5150956E082C748Ca837a5dFa0a7C10CA4697f9c',), ('_ceth_0x4B0181102A0112A2ef11AbEE5563bb4a3176c9d7',), ('_ceth_0xEE06A81a695750E71a662B51066F2c74CF4478a0',), ('_ceth_0xe047705117Eb07e712C3d684f5B18E74577e83aC',), ('_ceth_0x2D80f5F5328FdcB6ECeb7Cacf5DD8AEDaEC94e20',), ('_ceth_0x56b4F8C39E07D4d5D91692acf9d0F6d4d3493763',), ('_ceth_0x4B86e0295E7d32433FfA6411B82B4F4e56a581E1',), ('_ceth_0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3',), ('_ceth_0x63726dAe7C57d25e90ec829ce9a5C745Ffd984d3',), ('_ceth_0x000000000000d0151E748d25b766e77efe2A6c83',), ('_ceth_0x3D3D35bb9bEC23b06Ca00fe472b50E7A4c692C30',), ('_ceth_0x6fC13EACE26590B80cCCAB1ba5d51890577D83B2',), ('_ceth_0x618679dF9EfCd19694BB1daa8D00718Eacfa2883',), ('_ceth_0xD5525D397898e5502075Ea5E830d8914f6F0affe',), ('_ceth_0x84810bcF08744d5862B8181f12d17bfd57d3b078',), ('_ceth_0xf771733a465441437EcF64FF410e261516c7c5F3',), ('_ceth_0x7240aC91f01233BaAf8b064248E80feaA5912BA3',), ('_ceth_0x9F801c1F02AF03cC240546DadEf8e56Cd46EA2E9',), ('_ceth_0xF3312dc4c6D09D93C332844cF1ee58C268354c26',), ('_ceth_0xBF494F02EE3FdE1F20BEE6242bCe2d1ED0c15e47',), ('_ceth_0x226f7b842E0F0120b7E194D05432b3fd14773a9D',), ('_ceth_0xF5D0FefAaB749d8B14C27F0De60cC6e9e7f848d1',), ('_ceth_0x9043d4d51C9d2e31e3F169de4551E416970c27Ef',), ('_ceth_0x3fa400483487A489EC9b1dB29C4129063EEC4654',), ('_ceth_0xaeF4F02E31CdbF007f8D98da4aE365188A0E9eCC',), ('_ceth_0x3b62F3820e0B035cc4aD602dECe6d796BC325325',), ('_ceth_0xeea9aE787f3A620072d13b2cdC8cabFFb9c0aB96',), ('_ceth_0x4F4F0Ef7978737ce928BFF395529161b44e27ad9',), ('_ceth_0x20a68F9e34076b2dc15ce726d7eEbB83b694702d',), ('_ceth_0x3Ea50B7Ef6a7eaf7E966E2cb72b519C16557497c',), ('_ceth_0x5245C0249e5EEB2A0838266800471Fd32Adb1089',), ('_ceth_0x362bc847A3a9637d3af6624EeC853618a43ed7D2',), ('_ceth_0x7a5ce6abD131EA6B148a022CB76fc180ae3315A6',), ('_ceth_0x9c405acf8688AfB61B3197421cDeeC1A266c6839',), ('_ceth_0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5',), ('_ceth_0x639ae8F3EEd18690bF451229d14953a5A5627b72',), ('_ceth_0xBD4a858139B155219E2c8D10135003fdEf720b6B',), ('_ceth_0x1B8874BaceAAfba9eA194a625d12E8b270D77016',), ('_ceth_0x8669fA2B06829aa5FaBC47b5E5d0e66b85F1522E',), ('_ceth_0x7BCe667EF12023dc5f8577D015a2F09d99a5ef58',), ('_ceth_0x3031745E732dcE8fECccc94AcA13D5fa18F1012d',), ('_ceth_0x49D30E666E88c1a18F6cd9aDa5d138D0d3780858',), ('_ceth_0x8B39B70E39Aa811b69365398e0aACe9bee238AEb',), ('_ceth_0x270371C58D9D775ED73971Dd414656107384f235',), ('_ceth_0x67B6D479c7bB412C54e03dCA8E1Bc6740ce6b99C',), ('_ceth_0x2e2364966267B5D7D2cE6CD9A9B5bD19d9C7C6A9',), ('_ceth_0xdc9Ac3C20D1ed0B540dF9b1feDC10039Df13F99c',), ('_ceth_0xeDEec5691f23E4914cF0183A4196bBEb30d027a0',), ('_ceth_0xf21661D0D1d76d3ECb8e1B9F1c923DBfffAe4097',), ('_ceth_0xF9fbAefdE7112F78fA9BfE813341f0f49f888cB3',), ('_ceth_0x4C25Bdf026Ea05F32713F00f73Ca55857Fbf6342',), ('_ceth_0x9Ed8e7C9604790F7Ec589F99b94361d8AAB64E5E',), ('_ceth_0xeB953eDA0DC65e3246f43DC8fa13f35623bDd5eD',), ('_ceth_0x48C3399719B582dD63eB5AADf12A40B4C3f52FA2',), ('_ceth_0x72bbaE960006b950e4Dff6170b6266b1AEc0203c',), ('_ceth_0x87eDfFDe3E14c7a66c9b9724747a1C5696b742e6',), ('_ceth_0xa1069A93B0E44cc42bDceC90D010746d1b65a65C',), ('_ceth_0x3521c85C3000bff57Eac04489EB05BbD3193A531',), ('_ceth_0x56015BBE3C01fE05bc30A8a9a9Fd9A88917e7dB3',), ('_ceth_0x6A126b1A4d6A66370930Eca642eFF93d8e1be321',), ('_ceth_0x20c36f062a31865bED8a5B1e512D9a1A20AA333A',), ('_ceth_0x94939D55000B31B7808904a80aA7Bab05eF59Ed6',), ('_ceth_0x35Bd8472Ed2dA9EeD980e49B8b92CCBcf625aDbD',), ('_ceth_0x223B787a82d286BeC67b9f8068c1D679cA4cfB4a',), ('_ceth_0xf1F508c7C9f0d1b15a76fbA564eEf2d956220cf7',), ('_ceth_0x5a705745373a780814c379Ef17810630D529EFE0',), ('_ceth_0xa91c82B6933105C23298E631A0C1Be751D239162',), ('_ceth_0x3e780920601D61cEdb860fe9c4a90c9EA6A35E78',), ('_ceth_0x06B179e292f080871825beD5D722162fD96B4c95',), ('_ceth_0x31024A4C3e9aEeb256B825790F5cb7ac645e7cD5',), ('_ceth_0x76306F029f8F99EFFE509534037Ba7030999E3CF',), ('_ceth_0xF938424F7210f31dF2Aee3011291b658f872e91e',), ('_ceth_0xE270E54844fb2c8fe711f9Cf55C56AE425b01E1b',), ('_ceth_0x80c8C3dCfB854f9542567c8Dac3f44D709eBc1de',), ('_ceth_0x7Ef1081Ecc8b5B5B130656a41d4cE4f89dBBCC8c',), ('_ceth_0xD8fd31E40423a4CeF8e4a488793e87BB5b7D876D',), ('_ceth_0x5DE7Cc4BcBCa31c473F6D2F27825Cfb09cc0Bb16',), ('_ceth_0xd5281BB2d1eE94866B03A0fcCDd4e900c8Cb5091',), ('_ceth_0xAB7aaf9e485a3bc885985184ABE9FC6Aba727bD6',), ('_ceth_0x10994AA2FB8E6bA5D9FB2BC127FF228c4fe6167F',), ('_ceth_0x3371De12E8734c76F70479Dae3A9f3dC80CDCEaB',), ('_ceth_0x8D3E855f3f55109D473735aB76F753218400fe96',), ('_ceth_0xE95A203B1a91a908F9B9CE46459d101078c2c3cb',), ('_ceth_0xFc979087305A826c2B2a0056cFAbA50aad3E6439',), ('_ceth_0x846D152216146c77C81aF3A1657790eD8bA69281',), ('_ceth_0xC1Fa06e8596c3FA98cCd2113C38B6b60b6eFa00d',), ('_ceth_0x17eb50FDD2995696eE82912a80a9766fCBb0ECcA',), ('_ceth_0xf9FBE825BFB2bF3E387af0Dc18caC8d87F29DEa8',), ('_ceth_0x14Cfc7aEAA468E8C789785C39E0b753915AEB426',), ('_ceth_0xAAC1abdb4fB7a91a0e2E036dFacC45F708ed6a39',), ('_ceth_0xD23Ac27148aF6A2f339BD82D0e3CFF380b5093de',), ('_ceth_0xF0939011a9bb95c3B791f0cb546377Ed2693a574',), ('_ceth_0xb6c4267C4877BB0D6b1685Cfd85b0FBe82F105ec',), ('_ceth_0x5c4ac68aAc56eBe098D621Cd8CE9F43270Aaa355',), ('_ceth_0x4afb0AaC9b862946837b2444566B8a914D6d0d97',), ('_ceth_0x64fB96d0395f6bf105F35233911e3dF2C5bf4Ce8',), ('_ceth_0x08e411220e47e3Fc43BFb832186ABA95108F2861',), ('_ceth_0x41933422DC4a1cb8C822e06f12f7b52fA5E7E094',), ('_ceth_0x63b4f3e3fa4e438698CE330e365E831F7cCD1eF4',), ('_ceth_0x1dD80016e3d4ae146Ee2EBB484e8edD92dacC4ce',), ('_ceth_0x88bd6eFe33bc82860278c044eFA33364c6285032',), ('_ceth_0xe65cdB6479BaC1e22340E4E755fAE7E509EcD06c',), ('_ceth_0xF5B5eFc906513b4344EbAbCF47A04901f99F09f3',), ('_ceth_0x2aECCB42482cc64E087b6D2e5Da39f5A7A7001f8',), ('_ceth_0x44086035439E676c02D411880FcCb9837CE37c57',), ('_ceth_0xdacD69347dE42baBfAEcD09dC88958378780FB62',), ('_ceth_0xEa319e87Cf06203DAe107Dd8E5672175e3Ee976c',), ('_ceth_0xbEa98c05eEAe2f3bC8c3565Db7551Eb738c8CCAb',), ('_ceth_0x3F4E02741B155F5CE8D6190D294d4f916125B896',), ('_ceth_0x5Dc02Ea99285E17656b8350722694c35154DB1E8',), ('_ceth_0x0CDF9acd87E940837ff21BB40c9fd55F68bba059',), ('_ceth_0x208D174775dc39fe18B1b374972F77ddEc6c0F73',), ('_ceth_0x33c2DA7Fd5B125E629B3950f3c38d7f721D7B30D',), ('_ceth_0x3155BA85D5F96b2d030a4966AF206230e46849cb',), ('_ceth_0xB2FdD60AD80ca7bA89B9BAb3b5336c2601C020b4',), ('_ceth_0x7841B2A48D1F6e78ACeC359FEd6D874Eb8a0f63c',), ('_ceth_0x90b426067bE0b0FF5De257BC4dd6a4815Ea03b5f',), ('_ceth_0x88930072F583936F506CE1f1d5Fe69290C2D6A2a',), ('_ceth_0xeDB7b7842F7986a7f211d791e8F306C4Ce82Ba32',), ('_ceth_0x04B5E13000C6e9A3255Dc057091F3e3Eeee7b0f0',), ('_ceth_0x79126d32a86e6663F3aAac4527732d0701c1AE6c',), ('_ceth_0x81dBc1c8e40C3095071949Eda9800C2209a7279A',), ('_ceth_0xCb5f72d37685C3D5aD0bB5F982443BC8FcdF570E',), ('_ceth_0x04aB43d32D0172C76F5287B6619f0aa50aF89303',), ('_ceth_0xAa4e3edb11AFa93c41db59842b29de64b72E355B',), ('_ceth_0x70401dFD142A16dC7031c56E862Fc88Cb9537Ce0',), ('_ceth_0xC28e931814725BbEB9e670676FaBBCb694Fe7DF2',), ('_ceth_0xd0f05D3D4e4d1243Ac826d8c6171180c58eaa9BC',), ('_ceth_0x875773784Af8135eA0ef43b5a374AaD105c5D39e',), ('_ceth_0xCda2f16C6Aa895D533506B426AFF827b709c87F5',), ('_ceth_0x72F020f8f3E8fd9382705723Cd26380f8D0c66Bb',), ('_ceth_0xf65B5C5104c4faFD4b709d9D60a185eAE063276c',), ('_ceth_0x275f5Ad03be0Fa221B4C6649B8AeE09a42D9412A',), ('_ceth_0x435d0CA4d455209Dc9b53F09aCDb8D5D761df423',), ('_ceth_0xeB7e037140C1E2815F892830721778131BE0b0DF',), ('_ceth_0x30BCd71b8d21FE830e493b30e90befbA29de9114',), ('_ceth_0x1C9491865a1DE77C5b6e19d2E6a5F1D7a6F2b25F',), ('_ceth_0xfe9A29aB92522D14Fc65880d817214261D8479AE',), ('_ceth_0x4FE5851C9af07df9e5AD8217aFAE1ea72737Ebda',), ('_ceth_0xC96c1609A1a45CcC667B2b7FA6508e29617f7b69',), ('_ceth_0x07BE1eAd7AeBee544618Bdc688fA3CFf09857C32',), ('_ceth_0xea3983Fc6D0fbbC41fb6F6091f68F3e08894dC06',), ('_ceth_0x0aCe32f6E87Ac1457A5385f8eb0208F37263B415',), ('_ceth_0x44Fe65F8804C62d41ddcD6A48254988e5F88C2b5',), ('_ceth_0x7aa7305838d858F7089f86FC136d0816baaCB0d8',), ('_ceth_0x62359Ed7505Efc61FF1D56fEF82158CcaffA23D7',), ('_ceth_0x41D5D79431A913C4aE7d69a668ecdfE5fF9DFB68',), ('_ceth_0xA3c4Dc4A9cE2a6B40B57F25F8b50DeCc2c64deC2',), ('_ceth_0x774fb37e50DB4BF53b7C08e6B71007bf1F1D9a47',), ('_ceth_0x177BA0cac51bFC7eA24BAd39d81dcEFd59d74fAa',), ('_ceth_0x4c10BD19688B906665fBd53415f279F34b44ECE7',), ('_ceth_0xAa9D866666C2A3748d6B23Ff69E63E52f08d9AB4',), ('_ceth_0xea6412Fb370e8d1605E6aEeAA21aD07C3C7e9F24',), ('_ceth_0x87047986E8e4961c11d2EdcD94285E3A1331d97B',), ('_ceth_0xfE41168Fe09359c4b010E414c724c643dF4159A3',), ('_ceth_0x1614F18Fc94f47967A3Fbe5FfcD46d4e7Da3D787',), ('_ceth_0x45128CB743951121Fb70cb570c0784492732778A',), ('_ceth_0xaA62A002CE856D9fe1874E7f8558a38Fc1E49CCA',), ('_ceth_0xBae5F2D8a1299E5c4963eaff3312399253f27Ccb',), ('_ceth_0x1955d744F9435522Be508D1Ba60E3c12D0690B6A',), ('_ceth_0x9E78b8274e1D6a76a0dBbf90418894DF27cBCEb5',), ('_ceth_0xDc5864eDe28BD4405aa04d93E05A0531797D9D59',), ('_ceth_0x02Eca910CB3A7D43eBC7e8028652ed5C6b70259B',), ('_ceth_0x3166C570935a7D8554c8f4eA792ff965D2EFe1f2',), ('_ceth_0x9dDE0b1d39d0d2C6589Cde1BFeD3542d2a3C5b11',), ('_ceth_0x51d3e4C0b2c83E62f5d517D250b3e856897d2052',), ('_ceth_0xa1454f9c704AF96636F3A7532b9a04c411f85680',), ('_ceth_0xaB37e1358b639Fd877f015027Bb62d3ddAa7557E',), ('_ceth_0x3d3d5cCE38afb7a379B2C3175Ee56e2dC72CD7C8',), ('_ceth_0x5abFd418AdB35e89c68313574eB16BdfFc15e607',), ('_ceth_0xFbbE9b1142C699512545f47937Ee6fae0e4B0aA9',), ('_ceth_0x7B69D465c0F9FB22AffAE56aa86149973e9B0966',), ('_ceth_0xB26631c6dda06aD89B93C71400D25692de89c068',), ('_ceth_0x32C868F6318D6334B2250F323D914Bc2239E4EeE',), ('_ceth_0xE54f9E6Ab80ebc28515aF8b8233c1aeE6506a15E',), ('_ceth_0xF583Ae90b767183CA4c3F5087E2ee23892fabAd5',), ('_ceth_0xFAb5a05C933f1A2463E334E011992E897D56eF0a',), ('_ceth_0x3408B204d67BA2dBcA13b9C50e8a45701d8a1cA6',), ('_ceth_0x131C83801F4d9236f4416A0319cE7a02a76cAf94',), ('_ceth_0xCce41676a4624F4A1e33a787A59d6bF96e5067bc',), ('_ceth_0x155ff1A85F440EE0A382eA949f24CE4E0b751c65',), ('_ceth_0x1337DEF16F9B486fAEd0293eb623Dc8395dFE46a',), ('_ceth_0x16cAC1403377978644e78769Daa49d8f6B6CF565',), ('_ceth_0x9b53E429B0baDd98ef7F01F03702986c516a5715',), ('_ceth_0xE6F1966d04CfcB9cd1B1dC4E8256D8b501b11CbA',), ('_ceth_0x31B595e7cfDB624D10A3E7A562eD98c3567e3865',), ('_ceth_0x3C03b4EC9477809072FF9CC9292C9B25d4A8e6c6',), ('_ceth_0xD7B7d3C0bdA57723Fb54ab95Fd8F9EA033AF37f2',), ('_ceth_0x4f5fa8f2d12e5eB780f6082Dd656C565C48E0f24',), ('_ceth_0x3aaDA3e213aBf8529606924d8D1c55CbDc70Bf74',), ('_ceth_0x6CfB6dF56BbdB00226AeFfCdb2CD1FE8Da1ABdA7',), ('_ceth_0x19810559dF63f19cfE88923313250550eDADB743',), ('_ceth_0x56a980328AEE33AaBB540A02E002C8323326bf36',), ('_ceth_0xf0196985601598A35a48606b643FD2C34Fb861E1',), ('_ceth_0xb736bA66aAd83ADb2322D1f199Bfa32B3962f13C',), ('_ceth_0x888888888889C00c67689029D7856AAC1065eC11',), ('_ceth_0xDFe66B14D37C77F4E9b180cEb433d1b164f0281D',), ('_ceth_0xe13559cf6eDf84bD04bf679e251f285000B9305E',), ('_ceth_0x6E742E29395Cf5736c358538f0f1372AB3dFE731',), ('_ceth_0x432081eF9aa1b8503F8C7Be37E4bB158A0543Da9',), ('_ceth_0x6e0daDE58D2d89eBBe7aFc384e3E4f15b70b14D8',), ('_ceth_0xc3dD23A0a854b4f9aE80670f528094E9Eb607CCb',), ('_ceth_0x23B608675a2B2fB1890d3ABBd85c5775c51691d5',), ('_ceth_0x657B83A0336561C8f64389a6f5aDE675C04b0C3b',), ('_ceth_0x63B8b7d4A3EFD0735c4BFFBD95B332a55e4eB851',), ('_ceth_0xd5574177472Df0448AE56a9000E31Ff1611A3141',), ('_ceth_0xb526FD41360c98929006f3bDcBd16d55dE4b0069',), ('_ceth_0x072c46F392E729C1f0D92a307c2C6DBA06B5D078',), ('_ceth_0x3218A02F8F8B5c3894ce30EB255F10Bcba13E654',), ('_ceth_0x0020d80229877b495D2bf3269a4c13f6f1e1B9D3',), ('_ceth_0x74159651A992952e2bF340D7628459aA4593fc05',), ('_ceth_0x539F3615C1dBAfa0D008d87504667458acBd16Fa',), ('_ceth_0xEBd9D99A3982d547C5Bb4DB7E3b1F9F14b67Eb83',), ('_ceth_0x95b4eF2869eBD94BEb4eEE400a99824BF5DC325b',), ('_ceth_0x9248c485b0B80f76DA451f167A8db30F33C70907',), ('_ceth_0xf5dF66B06DFf95226F1e8834EEbe4006420D295F',), ('_ceth_0xD82BB924a1707950903e2C0a619824024e254cD1',), ('_ceth_0xdDce093888C8Dc636677Cd29BC6C68D35cFDAC33',), ('_ceth_0xBf131dCbE3436dab8a7c82D9C3666d652ca38eaB',), ('_ceth_0xF4c17Bc4979c1dc7b4CA50115358Dec58C67fD9d',), ('_ceth_0x8EF47555856f6Ce2E0cd7C36AeF4FAb317d2e2E2',), ('_ceth_0xf151980E7A781481709e8195744bF2399FB3Cba4',), ('_ceth_0x6006FC2a849fEdABa8330ce36F5133DE01F96189',), ('_ceth_0x76c5449F4950f6338A393F53CdA8b53B0cd3Ca3a',), ('_ceth_0xD69F306549e9d96f183B1AecA30B8f4353c2ECC3',), ('_ceth_0x2f4eb47A1b1F4488C71fc10e39a4aa56AF33Dd49',), ('_ceth_0xdB78f97B26757330bEDC2b4d9729c3E4C4560736',), ('_ceth_0xF5581dFeFD8Fb0e4aeC526bE659CFaB1f8c781dA',), ('_ceth_0xAec7e1f531Bb09115103C53ba76829910Ec48966',), ('_ceth_0xF136D7b0B7AE5b86D21E7B78DFA95375a7360f19',), ('_ceth_0x72630B1e3B42874bf335020Ba0249e3E9e47Bafc',), ('_ceth_0xbC81BF5B3173BCCDBE62dba5f5b695522aD63559',), ('_ceth_0x80a2AE356fc9ef4305676f7a3E2Ed04e12C33946',), ('_ceth_0x5F64Ab1544D28732F0A24F4713c2C8ec0dA089f0',), ('_ceth_0x4C6eC08CF3fc987c6C4BEB03184D335A2dFc4042',), ('_ceth_0x99f2B69ee2468C834A634FF50d930aE64e3e500f',), ('_ceth_0x358AA737e033F34df7c54306960a38d09AaBd523',), ('_ceth_0xB4ae194a0DCF1B4080b164C1d775ee06E0817305',), ('_ceth_0x5A43589fe110BB355Ba4A90a9C01476d87e68De8',), ('_ceth_0x8313f647C775664c2A2d381353Ea3b94dEF5465D',), ('_ceth_0x2Afcec14C53B7D4DEBFF766Cf0A934F1b3C01E7A',), ('_ceth_0x9Ea1Ae46C15a4164B74463Bc26f8aA3b0EeA2e6E',), ('_ceth_0x24E3794605C84E580EEA4972738D633E8a7127c8',), ('_ceth_0xB6Ca7399B4F9CA56FC27cBfF44F4d2e4Eef1fc81',), ('_ceth_0x92E187a03B6CD19CB6AF293ba17F2745Fd2357D5',), ('_ceth_0xD2F574637898526FCddfb3D487cc73c957Fa0268',), ('_ceth_0x7777777777697cFEECF846A76326dA79CC606517',), ('_ceth_0x801F90f81786dC72B4b9d51Ab613fbe99e5E4cCD',), ('_ceth_0xf33121A2209609cAdc7349AcC9c40E41CE21c730',), ('_ceth_0x76280AF9D18a868a0aF3dcA95b57DDE816c1aaf2',), ('_ceth_0x33F24Df0D3364ceeE080FE3B9C83e7eF1FA30843',), ('_ceth_0xB40554169eff159B80E563db17474e96D6F1EfCB',), ('_ceth_0x035dF12E0F3ac6671126525f1015E47D79dFEDDF',), ('_ceth_0xe09216F1d343Dd39D6Aa732a08036fee48555Af0',), ('_ceth_0x63D0eEa1D7C0d1e89d7e665708d7e8997C0a9eD6',), ('_ceth_0x89bD2E7e388fAB44AE88BEf4e1AD12b4F1E0911c',), ('_ceth_0xfa6f7881E52fDF912c4a285D78a3141B089cE859',), ('_ceth_0xB1e9157c2Fdcc5a856C8DA8b2d89b6C32b3c1229',), ('_ceth_0xC0bA369c8Db6eB3924965e5c4FD0b4C1B91e305F',), ('_ceth_0x69fa8e7F6bf1ca1fB0de61e1366f7412b827CC51',), ('_ceth_0xA3a7CBA479dF86C7F866280b90Ebc3B25f2371F0',), ('_ceth_0x3A9FfF453d50D4Ac52A6890647b823379ba36B9E',), ('_ceth_0x831091dA075665168E01898c6DAC004A867f1e1B',), ('_ceth_0xbfD815347d024F449886c171f78Fa5B8E6790811',), ('_ceth_0xEB58343b36C7528F23CAAe63a150240241310049',), ('_ceth_0xEC213F83defB583af3A000B1c0ada660b1902A0F',), ('_ceth_0x45080a6531d671DDFf20DB42f93792a489685e32',), ('_ceth_0xf680429328caaaCabee69b7A9FdB21a71419c063',), ('_ceth_0x412e5a36BDE71AA2c38e1c0E26BAAf7F2f0Bc24a',), ('_ceth_0xC888A0Ab4831A29e6cA432BaBf52E353D23Db3c2',), ('_ceth_0x0ED3cBE11e32A4db7D81b9055E464848B344fEE8',), ('_ceth_0xa58a4f5c4Bb043d2CC1E170613B74e767c94189B',), ('_ceth_0xFe2e637202056d30016725477c5da089Ab0A043A',), ('_ceth_0x1B8E12F839BD4e73A47adDF76cF7F0097d74c14C',), ('_ceth_0x2dC7b857772A4Ef7F9f39A88048F15374e4de161',), ('_ceth_0x06576eB3B212d605B797dC15523d9Dc9F4F66DB4',), ('_ceth_0xaD5Fe5B0B8eC8fF4565204990E4405B2Da117d8e',), ('_ceth_0xE1A0dA60F8802b49cC36313BafADe3F9e12342FD',), ('_ceth_0xeD761a603235b3Af7e8936875798D69d339a49B1',), ('_ceth_0xCf4f3b167c709952826e501e835a2C3A79785FbF',), ('_ceth_0xB9464ef80880c5aeA54C7324c0b8Dd6ca6d05A90',), ('_ceth_0x417fFdBc285dd2C4dC00937798ab901634137caA',), ('_ceth_0x59E9261255644c411AfDd00bD89162d09D862e38',), ('_ceth_0x73fa9181Cd0752A18DDFa285033df0323A804457',), ('_ceth_0x34612903Db071e888a4dADcaA416d3EE263a87b9',), ('_ceth_0x1Aa61c196E76805fcBe394eA00e4fFCEd24FC469',), ('_ceth_0x73374Ea518De7adDD4c2B624C0e8B113955ee041',), ('_ceth_0x29CbD0510EEc0327992CD6006e63F9Fa8E7f33B7',), ('_ceth_0x17EF75AA22dD5f6C2763b8304Ab24f40eE54D48a',), ('_ceth_0xD16c79c8A39D44B2F3eB45D2019cd6A42B03E2A9',), ('_ceth_0x814E351Ea87288501d52aE0E62b9F77B992f18aB',), ('_ceth_0x0DDe6F6e345bfd23f3F419F0DFe04E93143b44FB',), ('_ceth_0xC497e6Fe501A327C8eB5db50A3C6B110b9aC7d9E',), ('_ceth_0x30cF203b48edaA42c3B4918E955fED26Cd012A3F',), ('_ceth_0x6E765D26388A17A6e86c49A8E41DF3F58aBcd337',), ('_ceth_0x792348668c8d4B640Bb7C09D7127234d4C29053E',), ('_ceth_0x6768063279E2B185Dc0c972b97f11f231d0B45ad',), ('_ceth_0xb453f1f2EE776dAF2586501361c457dB70e1ca0F',), ('_ceth_0x8937041C8C52a78c25aa54051F6a9dAdA23D42A2',), ('_ceth_0x7b123f53421b1bF8533339BFBdc7C98aA94163db',), ('_ceth_0x84Bb947fcEdba6B9C7DCEad42dF07e113bb03007',), ('_ceth_0x3A880652F47bFaa771908C07Dd8673A787dAEd3A',), ('_ceth_0x2aF72850c504dDD3c1876C66a914cAee7Ff8a46A',), ('_ceth_0xf16e81dce15B08F326220742020379B855B87DF9',), ('_ceth_0x37E808F084101F75783612407e7C3f5F92d8ee3F',), ('_ceth_0xBD2F0Cd039E0BFcf88901C98c0bFAc5ab27566e3',), ('_ceth_0x697eF32B4a3F5a4C39dE1cB7563f24CA7BfC5947',), ('_ceth_0x3832d2F059E55934220881F831bE501D180671A7',), ('_ceth_0xC3a1fC184F159574628D06a883676A38710EBB8d',), ('_ceth_0x58EF3abAB72c6C365D4D0D8a70039752b9f32Bc9',), ('_ceth_0x4eCB692B0fEDeCD7B486b4c99044392784877E8C',), ('_ceth_0x7D85e23014F84E6E21d5663aCD8751bEF3562352',), ('_ceth_0xB2279B6769CFBa691416F00609b16244c0cF4b20',), ('_ceth_0x0563DCe613D559a47877fFD1593549fb9d3510D6',), ('_ceth_0x48f07301E9E29c3C38a80ae8d9ae771F224f1054',), ('_ceth_0x178c820f862B14f316509ec36b13123DA19A6054',), ('_ceth_0x26780078e07CC33f0323df626f42Fa92971561D4',), ('_ceth_0xcf9c692F7e62Af3c571D4173fd4ABf9A3E5330D0',), ('_ceth_0x8185Bc4757572Da2a610f887561c32298f1A5748',), ('_ceth_0xDDB3422497E61e13543BeA06989C0789117555c5',), ('_ceth_0x1A5F9352Af8aF974bFC03399e3767DF6370d82e4',), ('_ceth_0x2216e873ea4282EbEf7A02aC5aeA220bE6391A7C',), ('_ceth_0x173f8ee61C0FE712caE2a2fc8d5c0ccDDa57E6dA',), ('_ceth_0x202f1877e1DB1120Ca3e9A98C5D505e7f035C249',), ('_ceth_0x1443e7c1CCE72662545D94779120C59251447e91',), ('_ceth_0xFFc63b9146967A1ba33066fB057EE3722221aCf0',), ('_ceth_0x35101c731b1548B5e48bb23F99eDBc2f5c341935',), ('_ceth_0x6595b8fD9C920C81500dCa94e53Cdc712513Fb1f',), ('_ceth_0x1c7BBADc81E18F7177A95eb1593e5f5f35861B10',), ('_ceth_0x668DbF100635f593A3847c0bDaF21f0a09380188',), ('_ceth_0x2De27D3432d3188b53B02137E07B47896D347D45',), ('_ceth_0x14F0a12A43c36C49D4b403dD6e1A9B8222BE456C',), ('_ceth_0xC40AF1E4fEcFA05Ce6BAb79DcD8B373d2E436c4E',), ('_ceth_0x16886879d577815c3421Fbbee77c85c947caEFf6',), ('_ceth_0x4A65BA2E35de110536272C2cfE2B6d18C79d6F29',), ('_ceth_0x95b3497bBcCcc46a8F45F5Cf54b0878b39f8D96C',), ('_ceth_0xa0afAA285Ce85974c3C881256cB7F225e3A1178a',), ('_ceth_0x14Da7b27b2E0FedEfe0a664118b0c9bc68e2E9AF',), ('_ceth_0x07baC35846e5eD502aA91AdF6A9e7aA210F2DcbE',), ('_ceth_0x5bF4a9F93b9Cbe4eE8C211781e176994b57e569f',), ('_ceth_0x1cBb83EbcD552D5EBf8131eF8c9CD9d9BAB342bC',), ('_ceth_0x7BEF710a5759d197EC0Bf621c3Df802C2D60D848',), ('_ceth_0x9d695E811beb55c98b1B3668295e390d33dE5930',), ('_ceth_0x9048c33c7BaE0bbe9ad702b17B4453a83900D154',), ('_ceth_0x739763a258640919981F9bA610AE65492455bE53',), ('_ceth_0x4b7aD3a56810032782Afce12d7d27122bDb96efF',), ('_ceth_0x84679bc467DC6c2c40ab04538813AfF3796351f1',), ('_ceth_0x7f1F2D3dFa99678675ECE1C243d3f7bC3746db5D',), ('_ceth_0x93B2FfF814FCaEFFB01406e80B4Ecd89Ca6A021b',), ('_ceth_0xf6537FE0df7F0Cc0985Cf00792CC98249E73EFa0',), ('_ceth_0x037A54AaB062628C9Bbae1FDB1583c195585fe41',), ('_ceth_0x3593D125a4f7849a1B059E64F4517A86Dd60c95d',), ('_ceth_0x29B109625ac15BC4577d0b70ACB9e4E27F7C07E8',), ('_ceth_0x5cAf454Ba92e6F2c929DF14667Ee360eD9fD5b26',), ('_ceth_0x1DF6f1Bb7454E5E4BA3BcA882d3148FBf9b5697A',), ('_ceth_0xC12D1c73eE7DC3615BA4e37E4ABFdbDDFA38907E',), ('_ceth_0xF54025aF2dc86809Be1153c1F20D77ADB7e8ecF4',), ('_ceth_0x0E29e5AbbB5FD88e28b2d355774e73BD47dE3bcd',), ('_ceth_0xC237868a9c5729bdF3173dDDacaa336a0a5BB6e0',), ('_ceth_0x1a593D6fEaD832CeADa608Dd2fCeA7E4d79AF9B0',), ('_ceth_0x15F0EEDF9Ce24fc4b6826E590A8292CE5524a1DA',), ('_ceth_0xB6ff96B8A8d214544Ca0dBc9B33f7AD6503eFD32',), ('_ceth_0x6ce654aC973D326F89f0685E7459542641410eD9',), ('_ceth_0x92416e32042c9e67B0771A1b00Bcdf92EcB64950',), ('_ceth_0x26c8AFBBFE1EBaca03C2bB082E69D0476Bffe099',), ('_ceth_0xD7EFB00d12C2c13131FD319336Fdf952525dA2af',), ('_ceth_0xA1AFFfE3F4D611d252010E3EAf6f4D77088b0cd7',), ('_ceth_0xF406F7A9046793267bc276908778B29563323996',), ('_ceth_0x892A6f9dF0147e5f079b0993F486F9acA3c87881',), ('_ceth_0x5e06Ad5DCd057f82343C37eF8f3e73EB73cfbe69',), ('_ceth_0x6c972b70c533E2E045F333Ee28b9fFb8D717bE69',), ('_ceth_0x31bE30217989766215672E88eD449913e05bf0F5',), ('_ceth_0x86B0Aa51eB489585D88d2e671E5ee1b9e457Be60',), ('_ceth_0xc3771d47E2Ab5A519E2917E61e23078d0C05Ed7f',), ('_ceth_0x09fE5f0236F0Ea5D930197DCE254d77B04128075',), ('_ceth_0xc63CAAc1ad9954AD3bC8247Ab33fF233b0DD0FcD',), ('_ceth_0xCa3FE04C7Ee111F0bbb02C328c699226aCf9Fd33',), ('_ceth_0x14D1C83DF4dECEE9dEB14eE851f109f0101A6631',)}
```

Hehehe ... see uniswap and AMMs add the assets to the DB automatically. So with them it's not really a user-inserted asset, but still the thing hits. We need to think of a workaround here.

Reply:
So .. does this not fit in one line? Did not answer my question.

Reply:
No, It doesn't fit by a few chars